docs/sphinx/dev_guide/ci.rst

```diff
@@ -17,28 +17,24 @@ Continuous Integration (CI) Testing
                  viewed by clicking the appropriate link in the **checks** 
                  section of a GitHub pull request.
 
-The CI tools used by the RAJA project are:
+The RAJA project uses two CI tools to run tests:
 
   * **Azure Pipelines** runs builds and tests for Linux, Windows, and MacOS 
-    environments using recent versions of various compilers. While we do GPU 
-    builds for CUDA, HIP, and SYCL on Azure, RAJA tests are only run for 
-    CPU-only pipelines. Docker container images we use for the Linux testing
-    are maintained in the 
-    `RSE Ops Project <https://github.com/rse-ops/docker-images>`_. Please see 
-    the `RAJA Azure DevOps <https://dev.azure.com/llnl/RAJA>`_ project to learn 
-    more about our testing there.
-
-  * **GitLab** instances in the Collaboration Zone (CZ) of the Livermore 
-    Computing (LC) Center run builds and tests on LC HPC platforms using
+    environments using compilers in container image maintained in the
+    `RSE Ops Project <https://github.com/rse-ops/docker-images>`_.
+    While we do some GPU builds on Azure, RAJA tests are only run for CPU-only 
+    builds. The current set of builds run on Azure can be seen by looking at
+    the ``RAJA/azure-pipelines.yml`` and ``RAJA/Dockerfile`` files.
+
+  * **GitLab** instance in the Collaboration Zone (CZ) of the Livermore 
```

Comment:
```suggestion
  * **GitLab** instances in the Collaboration Zone (CZ) of the Livermore 
```

Reply:
There is only one GitLab instance.

Reply:
I was confused by `run`, but I see you changed it to `runs`.